sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Release History
 
-## 12.21.0b1 (Unreleased)
+## 12.21.0b1 (2024-12-10)
 
 ### Features Added
+- Added support for service version 2025-05-05.
+- Added support for `ShareFileClient` using the `NFS` protocol to `create_hard_link` 
+with an existing file by specifying full path starting from the root.
+- Added support for `ShareFileClient` using the `NFS` protocol to copy an existing file through the 
+`start_copy_from_url` API by specifying optional keywords `owner`, `group`, `file_mode`.
+- The following sync and async APIs no longer send request headers `x-ms-file-permission-key`, `x-ms-file-attributes`, 
+`x-ms-file-creation-time`, and `x-ms-file-last-write-time` by default. These headers have been optional
+in the REST API since the service version 2021-06-08.
```

Comment:
nit: remove "the" from "the service version" since in L6 we don't refer to it as "the service version" and this is consistent with the rest of the file

Reply:
Please also mention as the last sentence: "The default behavior of these APIs remains the same.". We don't want to scare people that we changed behavior here.